modules/Bio/EnsEMBL/Production/Pipeline/GPAD/LoadFile.pm

```diff
@@ -166,13 +166,18 @@ sub run {
       } elsif ($db =~ /RNAcentral/) {
         $self->log()->debug("Adding linkage to RNAcentral");
         $is_transcript = 1;
-        # Accession is the version with taxonomy id appended, e.g. URS0000007FBA_9606
-        # We store as URS0000007FBA, so need to remove everything from the _
-        # precursor_rna could be a list of accessions
-        my @precursor_rnas = split(",", $precursor_rna) if $precursor_rna;
-        foreach my $rna (@precursor_rnas) {
-          $rna =~ s/_[0-9]*//;
-          $db_object_id = $rna;
+        # For miRNA, we want the precursor(s), rather than the db ID,
```

Comment:
Sounds sensible, could you just add an example in the comments? for future reference. Thanks!